.github/workflows/docs.yml

```
@@ -0,0 +1,77 @@
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: apps/docs/dist
+
+      - name: 🚀 Deploy to GitHub Pages
+        if: steps.affected_build.outputs.should_deploy == 'true'
+        id: deployment
+        uses: actions/deploy-pages@v4
+
+      - name: ⏭️ Skip deploy (docs not affected)
+        if: steps.affected_build.outputs.should_deploy != 'true'
+        run: echo "Skipping deploy: @effectify/docs is not affected in this push."
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)